src/ir/context.rs

```diff
@@ -354,6 +354,8 @@ pub struct BindgenContext {
     /// This needs to be an std::HashMap because the cexpr API requires it.
     parsed_macros: StdHashMap<Vec<u8>, cexpr::expr::EvalResult>,
 
+    wrapper_ids: StdHashMap<String, TypeId>,
```

Comment:
Can you document this? It's not clear to me what you're trying to do here. Names can be namespaced in C++ too, which I think this wouldn't handle.

Reply:
Basically, I need a way to get the type ID from a name, e.g.

```c
typedef unsigned long MyType;
#define MY_CONST (MyType)123
```

So I need a way from  `"MyType"` to `c_ulong`. Maybe this is possible using an existing method, but I couldn't find one.